test/tftp/tftp_server_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'test_helper'
+require 'tftp/server'
+require 'tftp/tftp_plugin'
+
+module TftpGenericServerSuite
+  def setup
+    @rootdir = "/some/root"
+    @mac = "aa:bb:cc:dd:ee:ff"
+    @content = "file content"
+    FileUtils.stubs(:mkdir_p).returns(true)
```

Comment:
I don't think this stub is needed: mkdir_p call is made in set method; write_file takes care of creating of the dir now.